client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,20 +44,16 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Improvements in element templates</h4>
-          You can now configure a script task as a FEEL expression, define the called decision of a business rule task, and configure a form linked to a user task.
+          <h4>Enhanced BPMN ad-hoc sub-process support</h4>
+          You can now configure new ad-hoc sub-process properties such as output collection and implementation as a job worker.
         </li>
         <li>
-          <h4>Camunda FEEL editor intelligence</h4>
-          Our FEEL editor now supports <a href="https://docs.camunda.io/docs/components/modeler/feel/language-guide/feel-variables/#escape-variable-names">escaping variable names with backticks</a>.
+          <h4>Support for deployment using the new orchestration cluster REST API</h4>
```

Comment:
```suggestion
          <h4>Support for deployment using the new Orchestration Cluster REST API</h4>
```
Here the capitalization makes sense ([Orchestration Cluster is a specific Camunda component like Zeebe or Operate](https://docs.camunda.io/docs/next/reference/glossary/#orchestration-cluster))